apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -82,6 +82,7 @@ function ListBoxInline({ options, layout }) {
     scrollState = undefined,
     renderedCallback,
     toolbar = true,
+    isCollapsed = false,
```

Comment:
I was hoping we could enforce the toolbar to be open and enter the "selection state" from the outside, without having to introduce yet another option, but maybe it's not possible...

Let's discuss it tomorrow!